packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -61,7 +62,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
-        aria-describedby={ isTooltipVisible ? tooltipId : undefined }
+        aria-describedby={ (isTooltipVisible && !stopAriaDescribedBy) ? tooltipId : undefined }
```

Comment:
I didn't pay much attention to it until now but can someone help me undestand why there is a `aria-describedby` on tooltiphost? `aria-labelledby` and `aria-describedby` make sense only on elements that gain focus or are announced by ATs like Dialogs, Table headers etc. I think we added `aria-describedby` on TooltipHost as part of "display tooltip only on overflow" scenario. Right, @cschleiden ? For cases like Tooltip on button, attaching `aria-describedby` on host won't provide any help to screen readers since tooltiphost never gains focus.